css/main.css

```diff
@@ -1,3 +1,225 @@
 /* Add your CSS here */
+*{
 
+}
+
+body{
+    /* display: grid; */
+    /* grid-template-columns: ; */
+    /* min-width: auto; */
+    margin-top: 5rem;
+    font-family: "poppins", sans serif;
+  }
+
+.topnav{
+    display: flex;
+    flex-direction: row;
+     
+}
+
+#myLinks a {
+  text-decoration: none;
+  color: black;
+  float: right;
+  width:7em;
+  margin:10px auto;
+  font-size: 15px;
+}
+
+
+.container{
+  width:100%;
+  display: grid;
+  grid-template-areas: " left right";
+  position:absolute;
```

Comment:
This looks pretty wild on my screen. In general, using position:absolute for layout like this creates more problems than it solves.

Try resizing your browser, and choosing different views in Devtools Device Mode. 

https://developer.chrome.com/docs/devtools/device-mode/

What happens to your layout? 😱 😂